### DISCUSSION TITLE
Does AI have access to notes names?

### DISCUSSION THREAD

**Discussion by Atrocraz** [nicobrauchtgit/obsidian-Smart2Brain#172] [Q&A]
I'm just wondering, if LLM is able to use file names to create it's response. I was trying to refer to a daily note, but seems like LLM had no idea what i was talking about (i'm using mxbai-embed large for embedding and llama2 for chat).

**Comment by Leo310** [comment]
I know Its been some time, but it will be fixed in the next Release ;)
Wont take long anymore!
